assets/src/application/ui/input/InputWithLabel/style.scss

```diff
@@ -8,6 +10,10 @@
 			height: 42px;
 			margin: 0;
 			padding: 0;
+
+			@include max782px {
+				height: 40px;
```

Comment:
ideally, this value should derive from the input size, but that would mean that size needs to be passed to `InputWithLabel`, didn't want to slide through yak-shaving path here for now

Reply:
there was an issue that I tried to fix, but don't know which one

Reply:
updated: 			
![image](https://user-images.githubusercontent.com/1477453/83637401-3fb6e780-a5b0-11ea-824d-23e24445b06e.png)